openzwave-sys/src/value_classes/value_id.rs

```diff
@@ -24,17 +26,21 @@ pub enum ValueType {
     //ValueType_Max = ValueType_Raw // likely useless in Rust wrapper
 }
 
-pub enum ValueID {}
+#[repr(C)]
+pub struct ValueID {
+  pub home_id: uint32_t,
+  pub id: uint64_t
```

Comment:
nit: usually I use directly rust's u32 and u64 as these libc types do not bring much value.